pkg/cmd/pr/diff/diff.go

```diff
@@ -134,3 +142,26 @@ func isRemovalLine(dl string) bool {
 func validColorFlag(c string) bool {
 	return c == "auto" || c == "always" || c == "never"
 }
+
+func getDefaultPager() (string, error) {
+	getPagerCmd := git.GitCommand("config", "--get", "core.pager")
```

Comment:
Respecting git's `core.pager` is something that I feel warrants discussion. On one hand, this might feel like the right thing to do in `gh pr diff` since it feels so much like `git diff`. On the other hand, at no point do we actually invoke `git diff`, so I don't feel strongly that we should support _any_ of git's configuration around diffing and paging. In fact, instead of respecting `git config core.pager` or GIT_PAGER, I think we should only respect the PAGER environment variable.

@vilmibm Thoughts?

Reply:
I agree with @mislav that using `core.pager` for this can be cumbersome. Also, using `PAGER` env variable makes testeability of this feature easier, since we can mock that variable in tests!